NuGet.config

```diff
@@ -10,7 +10,10 @@
     <!--  Begin: Package sources from dotnet-extensions -->
     <!--  End: Package sources from dotnet-extensions -->
     <!--  Begin: Package sources from dotnet-runtime -->
+    <add key="darc-int-dotnet-runtime-08338fc" value="https://pkgs.dev.azure.com/dnceng/internal/_packaging/darc-int-dotnet-runtime-08338fca/nuget/v3/index.json" />
```

Comment:
Changes on this file can be removed as these versions are now available in dotnet-public.